### ISSUE TITLE
Create Preservica Logins for all DCS Admin Sets

### ISSUE THREAD

**Issue by motropuk** [yalelibrary/YUL-DC#1849]

**Comment by motropuk** [comment]
This is on hold until we have done full testing using the Music admin set login. Can then setup logins for the other repos

**Comment by motropuk** [comment]
I think we could start working on this now. It will take a while to complete, so best to start this now I think

**Comment by motropuk** [comment]
Worksheet https://docs.google.com/document/d/1TDF5pzpDygbs8vjOF5M6hHF17oP0HYtfCT27xIPhFAw/edit

**Comment by motropuk** [comment]
Waiting for DPS to review communication plan, before sending worksheet to stakeholders for review

**Comment by motropuk** [comment]
Plan approved by DPS

Will cc digitalpreservation@yale.edu on the messages to the unit liaisons, and DPS will be happy to jump in if there are any questions they can answer.

Once the tags for each are set DPS will put in the requests to get them setup. I will mention/remind units that DPS can change the security tags on items or parts of their collections in Preservica at their request.

DPS are also happy to support any changes that might come with the merging of BRBL and MSSA, they currently have different sets of tags and roles in Preservica.

**Comment by motropuk** [comment]
Emailed stakeholders to review spreadsheet. Gave them until EOD on Weds Apr 13th. So this ticket is waiting until then

**Comment by DraxIndustries79** [comment]
Check by Wed April 13

**Comment by motropuk** [comment]
Review of security tags by stakeholders is complete. Have given DPS the go ahead to create the new logins. Waiting on DPS to create these.

**Comment by motropuk** [comment]
We've got the credentials worked out for these accounts. The request is in to Library IT to create some additional Active Directory groups for access, then the request to add the service accounts to the relevant groups.

**Comment by motropuk** [comment]
Shared logins with @mikeapp to add to parameter stores

**Comment by mikeapp** [comment]
I think we should not do this on staging.

We are not going to be able to verify this is done until we have restarted management on the various instances.
